scripts/trainer.py

```diff
@@ -73,21 +37,22 @@ def go(self, model):
         logger.info("Decompressing bugs database")
         self.decompress_file("data/bugs.json")
 
-        if model == "defect":
-            # Train classifier for defect-vs-enhancement-vs-task.
-            self.train_defect_enhancement_task()
-        elif model == "component":
-            # Train classifier for the component of a bug.
-            self.train_component()
-        elif model == "regression":
-            # Train classifier for regression-vs-nonregression.
-            self.train_regression()
-        elif model == "tracking":
-            # Train classifier for tracking bugs.
-            self.train_tracking()
-        else:
-            # We shouldn't be here
-            raise Exception("valid_models is likely not up-to-date anymore")
+        log_messages = {
+            "component": "Training *component* model",
+            "defectenhancementtask": "Training *defect vs enhancement vs task* model",
+            "regression": "Training *regression vs non-regression* model",
+            "training": "Training *tracking* model",
+        }
+
+        if log_messages.get(model_name):
+            logger.info(log_messages[model_name])
```

Comment:
We could simply do `Training *{model}* name` (it's not exactly the same, but it works anyway and will require less code).